app/QueryserviceNamespace.php

```diff
@@ -27,6 +28,7 @@
  */
 class QueryserviceNamespace extends Model
 {
+    use HasFactory;
```

Comment:
There seems to be a reason why people use factories: in the test i need to fill `wiki_id` which isn't declared fillable. On a factory model _everything_ is fillable, which seems preferrable over making the field fillable, just so I can write a test.